condacolab.py

```diff
@@ -19,6 +19,8 @@
 from typing import Dict, AnyStr
 from urllib.request import urlopen
 from distutils.spawn import find_executable
+import ipywidgets as widgets
```

Comment:
There's a chance people don't have `ipywidgets` in their installer and I don't know if we should require it. What about using a `try / except` clause here to catch if it's not present, and then display the button conditionally?

Something like:

```
try:
  import ipywidgets
  HAS_IPYWIDGETS = True
except ImportError:
  HAS_IPYWIDGETS = False
```